buildkite/src/Jobs/Unit/Spec.dhall

```diff
@@ -0,0 +1,17 @@
+let S = ../../Lib/SelectFiles.dhall
+let D = S.PathPattern
+
+let JobSpec = ../../Pipeline/JobSpec.dhall
+
+in
+
+JobSpec::{
+  dirtyWhen = [
+    S::{ dir = Some [D.Lit "src/lib", D.Any], exts = Some ["ml"] },
+    S::{ dir = Some [D.Lit "src/nonconsensus", D.Any], exts = Some ["ml"] },
```

Comment:
I think these should change to `S.strictlyStart` and not restrict to ml (for example, if the `dune` file changes under these directories it could also invalidate unit tests.

Reply:
copy that